src/lib/storage/lqp_view.hpp

```diff
@@ -15,16 +15,13 @@ class AbstractLQPNode;
  */
 class LQPView {
  public:
-  LQPView(const std::shared_ptr<AbstractLQPNode>& lqp,
-          const std::unordered_map<ColumnID, std::vector<std::string>>& column_names);
+  LQPView(const std::shared_ptr<AbstractLQPNode>& lqp, const std::unordered_map<ColumnID, std::string>& column_names);
```

Comment:
Not really part of this PR, but should take no more than five minutes:
(1) Why is this a map anyway? Either we pass in nothing, or we give a list of the new column names. Partially renaming columns does not work, does it?
(2) As `column_names` will be copied anyway, it would be better to pass it in by value and `std::move` it in the initializer list. That would allow us to default it to `= {}`.

Reply:
- (1) It should also work with a plain `vector`. Indices would represent `ColumnID`s then. Renaming columns after `LQPView` was created does not make sense.
- (2) Great suggestion!

Reply:
- (1) It seems that we use a map instead of a vector to keep Hyrise's architecture consistent. `CreateView` inherits corresponding function signatures from `AbstractOperator`.

Reply:
- (2) I agree and adjusted the initializer.